pydis_site/settings.py

```diff
@@ -255,5 +254,17 @@
     "variables": {
         "primary": "#7289DA",
         "link": "$primary",
+        "navbar-height": "4.75rem",
+        "footer-padding": "1rem 1.5rem 1rem",
     }
 }
+
+# Which of our GitHub repos should be displayed on the front page, and in which order?
+HOMEPAGE_REPOS = [
+    "python-discord/site",
+    "python-discord/bot",
+    "python-discord/snekbox",
+    "python-discord/seasonalbot",
+    "python-discord/django-simple-bulma",
+    "python-discord/django-crispy-bulma",
+]
```

Comment:
i feel like this is abusing the settings.py a bit because we only need this on the front page view. maybe move it to a class variable for the index view?

Reply:
Okay, changed.